RecoBTag/Combined/plugins/DeepFlavourJetTagsProducer.cc

```diff
@@ -255,6 +261,13 @@ DeepFlavourJetTagsProducer::produce(edm::Event& iEvent, const edm::EventSetup& i
 			for(auto const& entry : toadd) {
 				nnout[entry.second] += nnout[entry.first];
 			}
+
+			//warn if the input is nan
+			for(auto entry : nnout) {
+				if(std::isnan(entry.second)) {
+					edm::LogWarning("ValueError") << "The NN output is nan for an event.";
```

Comment:
I have only one example, in wf 136.7611 in jenkins tests https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-23640/28813/runTheMatrix-results/136.7611_RunJetHT2016E_reminiaod+RunJetHT2016E_reminiaod+REMINIAOD_data2016_HIPM+HARVESTDR2_REMINIAOD_data2016_HIPM/step2_RunJetHT2016E_reminiaod+RunJetHT2016E_reminiaod+REMINIAOD_data2016_HIPM+HARVESTDR2_REMINIAOD_data2016_HIPM.log

it looks like once the input is NaN, the output will also be a NaN.
Instead of many LogWarning messages (likely always 5 for each output + at least one per NaN input)  for essentially the same origin, please reformat the warning to make just one summary message.